fdbserver/ptxn/TLogInterface.h

```diff
@@ -100,9 +116,17 @@ struct TLogPeekReply {
 	// StringRef referring the serialized mutation data, see TLogStorageServerPeekSerializer
 	StringRef data;
 
+	Version end;
+	Optional<Version> popped;
+	Version maxKnownVersion;
+	Version minKnownCommittedVersion;
+	Optional<Version> begin;
```

Comment:
Better add comments for these, which can be confusing for readers.

Reply:
Will do that when I implement pop. Definitions of those are tricky and are related to other concepts in pop. Here I just put some placeholder from the old code.